examples/integrations/neo4j/pom.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2017, 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Please fix the copyright year, this should not contain `2017`

Reply:
done